content/logs/index.md

```diff
@@ -70,13 +68,13 @@ You might not find any log section in your yaml files during the private beta, h
 
 ## Custom log collection
 
-The Datadog Agent can collect logs from files or the network (TCP or UDP) and forward them to Datadog. To configure this, rename the `custom-logs.yaml.example` file to `custom-logs.yaml` (or to some other unused `*.yaml` name) and set these options:
+The Datadog Agent can collect logs from files or the network (TCP or UDP) and forward them to Datadog. To configure this, rename the `custom-logs.yaml.example` file to `custom-logs.yaml` and set these options:
```

Comment:
we must not reference any "custom-logs.yaml.example" as it does not existe.

We should instead say "configure a custom yaml file with the below options":